packages/stream-tile-handler/src/__tests__/tile-document.test.ts

```diff
@@ -185,27 +185,35 @@ describe('TileDocumentHandler', () => {
     })
 
     it('makes genesis record correctly', async () => {
-        const record1 = await TileDocument.makeGenesis(context.api, RECORDS.genesis.data, { deterministic: true })
-        expect(record1).toBeDefined()
+        const record = await TileDocument.makeGenesis(context.api, RECORDS.genesis.data)
```

Comment:
don't worry about changing this now, but for future reference if you're ever working on an old test and see a reference to "records" on a test you're changing anyway it'd be nice to use that opportunity to change it to "commits" which is the current terminology for these things